meshroom/ui/qml/ImageGallery/ImageGallery.qml

```diff
@@ -39,6 +39,13 @@ Panel {
         property variant viewpoints: currentCameraInit ? currentCameraInit.attribute('viewpoints').value : undefined
         property variant intrinsics: currentCameraInit ? currentCameraInit.attribute('intrinsics').value : undefined
         property bool readOnly: root.readOnly || displayHDR.checked
+
+        onViewpointsChanged: {
```

Comment:
I don't think this callback is necessary, the logic should be in `onCountChanged` (to keep things tidy and avoid relying on QML computing order).

Also this block creates a QML warning: `ImageGallery.qml:45: TypeError: Cannot read property 'count' of undefined`.

Reply:
Turns out the fix can actually be a lot simpler than that 😄 Thanks a lot for your help @mugulmd!